packages/ignite-cli/src/lib/exitCodes.js

```diff
@@ -65,5 +65,15 @@ module.exports = {
   /**
    * One of the global dependencies are not met.
    */
-  INVALID_GLOBAL_DEPENDENCY: 12
+  INVALID_GLOBAL_DEPENDENCY: 12,
+
+  /**
+   * There is no 13th floor.. only Zuul
+   */
+  TRISKAIDEKAPHOBIA_DETECTED: 13,
```

Comment:
I'm really really disappointed this isn't in our current code base @GantMan ... sneak it in your next PR!